.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+﻿## 変更点
```

Comment:
Codacy found an issue: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/gh/hrntsm/STBReader/pullRequest?prid=6384839)